tests/layer_tests/tensorflow_tests/test_tf_Conv2D.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from common.tf_layer_test_class import CommonTFLayerTest
+
+import logging
+
+# Testing operation Conv2D
+# Documentation: https://www.tensorflow.org/api_docs/python/tf/raw_ops/Conv2D
+
+class TestConv2D(CommonTFLayerTest):
+    # input_shape - should be an array, shape of input tensor in format [batch, height, width, channels]
+    # input_filter - should be an array, defines a filter
+    # input_strides - should be an array, defines strides of a sliding window to use
+    # input_padding - should be a string, defines padding algorithm
+    # ir_version - common parameter
+    # use_new_frontend - common parameter
+    def create_conv2d_placeholder_const_net(self, input_shape, input_filter, input_strides, input_padding, dilations, ir_version, use_new_frontend):
+        """
+            Tensorflow net                  IR net
+
+            Placeholder->Conv2D    =>       Placeholder->Transpose->Convolution--------------------->Reshape
```

Comment:
nit: Placeholder->Transpose->Convolution->Transpose?
Please correct this in the next PR

Reply:
Fixed. But I'm sure I saw such workflow in IR's... Never mind...